src/components/Dropdown/Dropdown.jsx

```diff
@@ -229,7 +229,7 @@ const Dropdown = ({
         Menu,
         ClearIndicator,
         Input,
-        ...(finalOptionRenderer && { Option }),
+        Option,
```

Comment:
why we removed finalOptionRenderer ?

Reply:
Here all custom element renders are specified. Previously the logic was like this: if `optionRenderer` has been passed then use this custom function to render option, otherwise use default renderer. Now it's always using custom renderer, cause it's the only option to add Tooltip wrapper for the option. And now `options.jsx` option-renderer is the entry point and it's responsible for deciding how specific option should be rendered